dpetryshintl/sonic-buildimage
# SWI-3605 [Barefoot] Upgrade docker-syncd-bfn to buster

**- Why I did it**

To upgrade bfn syncd to buster

**- How I did it**

Change docker-syncd-bfn docker file.
Update BFN SDK to buster

**- How to verify it**

Check syncd is running on buster (cat /etc/debian-version).
Ensure that all dockers and ip interfaces are up.
